Makefile.work

```diff
@@ -164,7 +164,7 @@ SONIC_BUILD_INSTRUCTION :=  make \
                            SHUTDOWN_BGP_ON_START=$(SHUTDOWN_BGP_ON_START) \
                            SONIC_ENABLE_PFCWD_ON_START=$(ENABLE_PFCWD_ON_START) \
                            SONIC_ENABLE_SYNCD_RPC=$(ENABLE_SYNCD_RPC) \
-                           SONIC_INSTALL_DEBUG_TOOLS=$(INSTALL_DEBUG_TOOLS) \
+                           INSTALL_DEBUG_TOOLS=$(INSTALL_DEBUG_TOOLS) \
```

Comment:
this should be SONIC_INSTALL_DEBUG_TOOLS, check line 106 in slave.mk

Reply:
Thanks for pointing this out. I missed this one file change while pushing my changes.